src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -2512,7 +2512,7 @@ private DataObject Copy(NormalizedSnapshotSpanCollection spans)
             {
                 var text = GetText(spans);
                 var blocks = GetTextBlocks(spans);
-                var rtf = _rtfBuilderService.GenerateRtf(spans, TextView);
+                var rtf = ((IRtfBuilderService2)_rtfBuilderService).GenerateRtf(spans, CancellationToken.None);
```

Comment:
@dpoeschl Maybe we can do something similar to [this](http://index/#Microsoft.VisualStudio.Platform.Editor/EditorOperations.cs,3262)?